docker-compose.yml

```diff
@@ -9,6 +31,10 @@ services:
     user: postgres
     volumes:
       - pgdata:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
```

Comment:
This is a smell. Why do we need to expose the database to the host OS?

Reply:
Coturn needs to access the database. Is there another better way of inter container communication?

Reply:
Containers on the same network can reach one another. The _ports_ configuration maps the container port to the port on the host OS. So in this case, that’s `HOST:CONTAINER` which maps port 5432 from the container to the same port on the host OS. Will software on your local machine need to access the database directly?